### PR TITLE
[stable/redmine] Option to create a custom service account

### PR DESCRIPTION
**Pull request by crmarques**
#### What this PR does / why we need it:

Add option to create a custom service account.

#### Special notes for your reviewer:

It does not change chart's default behavior.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ ] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [ ] Chart Version bumped
- [ ] Variables are documented in the README.md
- [ ] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

**Comment by crmarques** [comment]
@carrodher, could you review this PR?

**Comment by crmarques** [comment]
@javsalgar @tompizmor, could you review this PR? Thanks!

File: stable/redmine/values.yaml [hunk 1]
```diff
@@ -302,6 +302,16 @@ resources: {}
   #   cpu: "2"
   #   memory: "1G"
 
+## Pods Service Account
+## ref: https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/
+serviceAccount:
+  ## Specifies whether a ServiceAccount should be created
+  ##
+  create: false
```
Comment (davidkarlsen) [inline stable/redmine/values.yaml]:
```suggestion
  create: true
```
https://helm.sh/docs/topics/chart_best_practices/rbac/

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

**Review by juan131 — APPROVED** [review]
Sorry I missed this PR! But it looks great to me
Approving it

**Comment by crmarques** [comment]
@juan131, Is it ok to proceed? =D

**Pull request merged**